cypress/e2e/brewery/ScenarioViewDashboard.cy.js

```diff
@@ -93,4 +81,31 @@ describe('Scenario view PowerBI report', () => {
     ScenarioParameters.discard();
     Scenarios.checkIfReportIsUnsynced(true);
   });
+
+  it('If the back end returns a start time as null, out-of-synch is never displayed', () => {
+    const CURRENCY_VALUE_TO_UPDATE = 97779;
+    stub.setScenarioRunOptions({ startTime: null });
+
+    ScenarioParameters.expandParametersAccordion();
+    Scenarios.checkIfReportIsUnsynced(false);
+    Scenarios.getScenarioBackdrop(10).should('not.be.visible');
+    Scenarios.getDashboardAccordionLogsDownloadButton().should('not.exist');
+    Scenarios.checkIfReportIsUnsynced(false);
```

Comment:
This second check `Scenarios.checkIfReportIsUnsynced(false);` is not necessary because there have been no user interactions since the previous checks 3 lines above, only other checks